spec/listing/listing_spec.js

```diff
@@ -154,48 +141,37 @@ describe("Listing Plugin", function() {
       it("should purge the cache fo the removed model", function() {
         expect(listing.cache.size()).toEqual(1);
       });
-
     }
 
     describe("collection is removed from", function() {
-
       beforeEach(function() {
-        collection.reset([ { name: "Bob"}, { name: "Joe"} ]);
+        collection.reset([ { name: "Bob" }, { name: "Joe" } ]);
```

Comment:
Same array comment here, also is a few other places in this diff.

Reply:
I'll look into updating our EsLint config.